### PR TITLE
Rename StandAloneConsensus to OneCopyConsensus

### PR DESCRIPTION
**Pull request by JackieTien97**
Rename StandAloneConsensus to OneCopyConsensus

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeStartupCheck.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ private void checkGlobalConfig() throws ConfigurationException {
 
     // When the data region consensus protocol is set to StandAlone,
```
Comment (HTHou) [inline confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeStartupCheck.java:64]:
```suggestion
    // When the data region consensus protocol is set to ONE_COPY_CONSENSUS,
```

File: confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeStartupCheck.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ private void checkGlobalConfig() throws ConfigurationException {
 
     // When the schema region consensus protocol is set to StandAlone,
```
Comment (HTHou) [inline confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeStartupCheck.java:74]:
```suggestion
    // When the schema region consensus protocol is set to ONE_COPY_CONSENSUS,
```

File: consensus/src/main/java/org/apache/iotdb/consensus/ConsensusFactory.java [hunk 3]
```diff
@@ -32,8 +32,8 @@ public class ConsensusFactory {
   public static final String CONSTRUCT_FAILED_MSG =
       "Construct consensusImpl failed, Please check your consensus className %s";
 
-  public static final String StandAloneConsensus =
-      "org.apache.iotdb.consensus.standalone.StandAloneConsensus";
+  public static final String ONE_COPY_CONSENSUS =
+      "org.apache.iotdb.consensus.onecopy.OneCopyConsensus";
   public static final String RatisConsensus = "org.apache.iotdb.consensus.ratis.RatisConsensus";
```
Comment (xingtanzjr) [inline consensus/src/main/java/org/apache/iotdb/consensus/ConsensusFactory.java:37]: Rename these two variables in passing ?

File: consensus/src/main/java/org/apache/iotdb/consensus/ConsensusFactory.java [hunk 4]
```diff
@@ -32,8 +32,8 @@ public class ConsensusFactory {
...
   public static final String MultiLeaderConsensus =
```
Comment (xingtanzjr) [inline consensus/src/main/java/org/apache/iotdb/consensus/ConsensusFactory.java:38]: Rename these two variables in passing ?

Comment (xingtanzjr) [reply consensus/src/main/java/org/apache/iotdb/consensus/ConsensusFactory.java:38]: We can change it in next PR

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**